set.archived.sh

```diff
@@ -28,7 +28,7 @@ echo "** Updating ${dataset_name}.analysis_archived table. **"
 # Child studies <run_accession>
 #curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d 'result=analysis&query=study_accession%3D%22PRJEB43947%22%20OR%20study_accession%3D%22PRJEB45554%22%20OR%20study_accession%3D%22PRJEB45619%22%20OR%20study_accession%3D%22PRJEB55349%22%20OR%20study_accession%3D%22PRJEB55355%22%20OR%20study_accession%3D%22PRJEB55356%22%20OR%20study_accession%3D%22PRJEB57992%22%20OR%20study_accession%3D%22PRJEB57993%22%20OR%20study_accession%3D%22PRJEB57995%22%20OR%20study_accession%3D%22PRJEB59443%22%20OR%20study_accession%3D%22PRJEB59444%22%20OR%20study_accession%3D%22PRJEB59445%22%20OR%20study_accession%3D%22PRJEB61667%22%20OR%20study_accession%3D%22PRJEB61668%22%20OR%20study_accession%3D%22PRJEB61669%22&fields=ALL&format=tsv&limit=10' \
```

Comment:
Nitpicking: it might be better to keep the comments from 11 to 18 and the debug code on line 25 & 29 updated as well.

Reply:
I've updated the comments